frame/chainrelay/eth/relay/src/mock_headers/utils.rs

```diff
@@ -0,0 +1,133 @@
+//! utils for parsing eth headers
+use std::num::ParseIntError;
+use std::str::FromStr;
+
+use eth_primitives::{Bloom, EthAddress, H64};
+use hex_literal::hex;
+use json::object::Object;
+use rustc_hex::FromHex;
+
+use crate::*;
+
+pub use eth_primitives::receipt::LogEntry;
+pub use json::JsonValue;
+
+/// hex
+macro_rules! hex {
```

Comment:
Just use `<[u8: $len]>::from_hex($array_like)`

Reply:
I'll replace the handwrite macro to rustc_hex